cumulusci/tasks/bulkdata/step.py

```diff
@@ -420,10 +420,12 @@ def load_records(self, records):
         def _convert(rec):
             result = dict(zip(self.fields, rec))
             for boolean_field in self.boolean_fields:
-                # This is clumsy but required since Booleans are stored as.
-                result[boolean_field] = (
-                    result[boolean_field] or "false"
-                ).lower() == "true"
+                try:
+                    result[boolean_field] = process_bool_arg(
+                        result[boolean_field] or False
```

Comment:
What purpose is served by the `or False`?

Reply:
It avoids passing None given that accepting None is just a quirk of process_bool_arg() and not a feature we'd prefer to support forever.